src/staticanalysis/bot/static_analysis_bot/__init__.py

```diff
@@ -165,6 +167,10 @@ def is_third_party(self):
         '''
         Is this issue in a third party path ?
         '''
+        # TODO: download the third party file from hgweb and use it here
```

Comment:
I woud remove this TODO, for "remote" we rely on try to ignore third-party files, so we will always return false here (until we totally remove the check).

Reply:
Good point :+1: